### PR TITLE
Job role updates

### PR DESCRIPTION
**Pull request by natstar93**
**Description**
- Updates job role descriptions
- Adds end dates
- Changes ordering of roles

**QA**
- Please check for typos and general readability :P

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,41 +2,43 @@
...
-#### Senior Web Engineer (Apr 2020 – Present)
+#### Senior Web Engineer (Apr 2020 – March 2023)
 - Taking the lead on multiple deliverables requiring collaboration within the team and with other teams 
 - Identifying improvements to testing 
 - Researching and implementing new libraries and new tech (React hooks, React Testing Library)
 - Running front end learning sessions and pairing on TDD with more junior team members
 
 Achievements
-- Helped an API focussed team design new endpoints suitable for our use cases and rewrote parts of the integration test suite to mock the new responses
+- Led two teams to implement white labelling across 4 frontend applications, finding configuration solutions to heavily customised projects
```
Comment (duskyshelf) [inline README.md]: This sentence read strange to me. "finding configuration solutions to heavily customised projects" particularly

File: README.md [hunk 2]
```diff
@@ -2,41 +2,43 @@
...
 - Took the lead on both a delay compensation project and a redesign of the site's bookings view, both of which have received very positive feedback and were delivered in time for the site migration
+- Helped an API focussed team design new endpoints suitable for our use cases and rewrote parts of the integration test suite to mock the new responses
```
Comment (duskyshelf) [inline README.md]: Can probably de-fluff this one

**Review by duskyshelf — COMMENTED** [review]
Looks good.

File: README.md (hunk 2 above)
Comment (natstar93) [reply README.md]: So fluffy

File: README.md [hunk 3]
```diff
@@ -22,34 +22,35 @@ I am looking for a role that focusses on delivery and maintenance of high qualit
...
+- Provided empathetic support to team members by actively listening and addressing their concerns and challenges
 
 #### Developer (Oct 2017 – Mar 2020)
-- Building greenfield React applications and contributing to a new stateful login & customer migration form. 
-- Fetching data from external APIs using Node and exposing endpoints to the client side. 
-- Delivering features end to end, from planning and estimating through to production deployment.
-- Helping organise meetups at Trainline and coaching at Code First Girls and Codebar.
+- Building greenfield React applications and contributing to a new stateful login & customer migration form
+- Fetching data from external APIs using Node and exposing endpoints to the client side
+- Delivering features end to end, from planning and estimating through to production deploymen
```
Comment (duskyshelf) [inline README.md]: typo

**Pull request merged**